src/Microsoft.ML.ImageAnalytics/doc.xml

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="ImageGrayscalingEstimator">
+      <summary>
+        Converts the images to grayscale.
+      </summary>
+      <remarks>
+        <format type="text/markdown">
+            <![CDATA[ 
+
+          ## Remarks 
+          The images might be converted to grayscale to reduce the complexity of the model.
+          The grayed out images contain less information to process than the colored images.
+          Another use case for converting to grayscale is to generate new images out of the existing ones, so you can have a larger dataset,
+          a technique known as <a href="http://www.stat.harvard.edu/Faculty_Content/meng/JCGS01.pdf"> data augmentation</a>.
+          
+          For end-to-end image processing pipelines, and scenarios in your applications, see the 
+          <a href="https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started">examples in the machinelearning-samples github repository.</a>
+        ]]></format>
+        <seealso cref="ImageEstimatorsCatalog" />
+        <seealso cref="ImageLoadingEstimator"/>
+    </remarks>
+    </member>
+
+    <member name="ImageLoadingEstimator">
+      <summary>
+        Loads the images from a given folder into memory as an <see cref="ImageType"/>
+      </summary>
+      <remarks>
+        The images to load are present in the folder specfied as imageFolder. They get loaded in memory as a <see cref="System.Drawing.Bitmap" /> type. 
+        Loading the images in memory is the first step of almost every pipeline that does image processing, and further analysis on images. 
+        The images to load need to be in the formats supported by <see cref="System.Drawing.Bitmap" />.
+
+        For end-to-end image processing pipelines, and scenarios in your applications, see the
```

Comment:
remove comment - For end-to-end image processing pipelines and scenarios in your application, see the...